site/docs/spec.md

```diff
@@ -486,26 +486,29 @@ Lists must use the [3-level representation](https://github.com/apache/parquet-fo
-| **`timestamptz`**  | `struct`    | We should add this to ORC’s type model (ORC-294).                                       |
-| **`string`**       | `string`    | ORC `varchar` and `char` would also map to **`string`**.                                |
-| **`uuid`**         | `binary`    |                                                                                         |
-| **`fixed(L)`**     | `binary`    | The length would not be checked by the ORC reader and should be checked by the adapter. |
-| **`binary`**       | `binary`    |                                                                                         |
-| **`struct`**       | `struct`    | ORC `uniontype` would also map to **`struct`**.                                         |
-| **`list`**         | `array`     |                                                                                         |
-| **`map`**          | `map`       |                                                                                         |
+| Type               | ORC type            | ORC type attributes                                  | Notes                                                                                   |
+|--------------------|---------------------|------------------------------------------------------|-----------------------------------------------------------------------------------------|
+| **`boolean`**      | `boolean`           |                                                      |                                                                                         |
+| **`int`**          | `int`               |                                                      | ORC `tinyint` and `smallint` would also map to **`int`**.                               |
+| **`long`**         | `long`              |                                                      |                                                                                         |
+| **`float`**        | `float`             |                                                      |                                                                                         |
+| **`double`**       | `double`            |                                                      |                                                                                         |
+| **`decimal(P,S)`** | `decimal`           |                                                      |                                                                                         |
+| **`date`**         | `date`              |                                                      |                                                                                         |
+| **`time`**         | `long`              | `iceberg.long-type`=`DATE`                           | Stores microseconds from midnight.                                                      |
```

Comment:
Looks like this should be `time` and not `date`?

Reply:
My bad. Fixed.